### PR TITLE
Create 0702.tex for a solution of exercise 2 of APE7

### PR DESCRIPTION
**Pull request by anckaertv**
I have written the results obtained in the exercise session on Latex, with some explanation. I am just not sure how the sequential equilibrium needs to be written, I guess that's only one sequential equilibrium taking account of the randomised strategy + d_2 but I don't know what's the correct notation. For the other points, I have tried to explain the best that I could... Tell me if I need to change anything! 

Maybe a tikzpicutre for the last point could be helpful but when I tried to add the lambda, beta, and so on it did not looked good

### CODE REVIEW

File: exercises/APE7/0702.tex [hunk 1]
```diff
@@ -0,0 +1,53 @@
+\paragraph{a.} Let's first define what a belief vector is : for $\pi \in  \times_{s \in S} \Delta Y_s $, $i \in N$, $s \in S_i$, and $x \in Y_s$, $\pi_s(x)$ is the probability assigned by player $i$ to be at node $x$ given the fact that $i$ knows his information state $s$. ($X$ the nodes, $Y_s\in X$ is the set of nodes that have information $s$). \\
```
Comment (blegat) [inline exercises/APE7/0702.tex]: Don't use `\\` outside a `{tabular}`. Leave a blank line if you want to create a new paragraph

File: exercises/APE7/0702.tex [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+\begin{itemize}
+\item $\pi_{1.1}= 1$ 
+\item $\pi_{2.2.a_1} = \frac{0.6}{0.6+0.4 \cdot 0.25} = \frac{6}{7}$
+\item $\pi_{2.2.b_1} = \frac{0.4\cdot 0.25}{0.6+0.4 \cdot 0.25} = \frac{1}{7}$ 
+\item $\pi_{2.3}     = 1 $
+\end{itemize}
+$\pi_{1.1}=\pi_{2.3} = 1$ because there is only one possible way to be at 1.1 (or 2.3) knowing that the information state is 1.1 (or 2.3).
+
+\paragraph{b.} The sequential value of player $i$ for the move $d$ and information state $s$ $$ U_i(d| s, (\tau, \pi) ) = \sum_{x \in Y_s} \pi_s(x) U_i((\tau_{-i}, d), x)$$
+Meaning that we will only care about nodes x with information $s$. \\
```
Comment (blegat) [inline exercises/APE7/0702.tex]: Same here about `\\`

File: exercises/APE7/0702.tex [hunk 3]
```diff
@@ -0,0 +1,53 @@
+\paragraph{a.} Let's first define what a belief vector is : for $\pi \in  \times_{s \in S} \Delta Y_s $, $i \in N$, $s \in S_i$, and $x \in Y_s$, $\pi_s(x)$ is the probability assigned by player $i$ to be at node $x$ given the fact that $i$ knows his information state $s$. ($X$ the nodes, $Y_s\in X$ is the set of nodes that have information $s$). \\
+We can calculate it we Baye's formula $$ \pi_s(y) = \frac{P(y | \tau, x^0)}{\sum_{x \in Y_s} P(x | \tau, x^0)}. $$
```
Comment (blegat) [inline exercises/APE7/0702.tex]: "Baye's" -> Bayes'

File: exercises/APE7/0702.tex [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Here there is four information states: 1.1, 2.2 via $\text{a}_1$, 2.2 via $\text{b}_1$ and 2.3: 
+
+\begin{itemize}
+\item $\pi_{1.1}= 1$ 
+\item $\pi_{2.2.a_1} = \frac{0.6}{0.6+0.4 \cdot 0.25} = \frac{6}{7}$
```
Comment (blegat) [inline exercises/APE7/0702.tex]: Write it `\pi_{2.2}(2.2.a_1)`

File: exercises/APE7/0702.tex [hunk 5]
```diff
@@ -0,0 +1,53 @@
+\paragraph{a.} Let's first define what a belief vector is : for $\pi \in  \times_{s \in S} \Delta Y_s $, $i \in N$, $s \in S_i$, and $x \in Y_s$, $\pi_s(x)$ is the probability assigned by player $i$ to be at node $x$ given the fact that $i$ knows his information state $s$. ($X$ the nodes, $Y_s\in X$ is the set of nodes that have information $s$). \\
+We can calculate it we Baye's formula $$ \pi_s(y) = \frac{P(y | \tau, x^0)}{\sum_{x \in Y_s} P(x | \tau, x^0)}. $$
+
+Here there is four information states: 1.1, 2.2 via $\text{a}_1$, 2.2 via $\text{b}_1$ and 2.3:
```
Comment (blegat) [inline exercises/APE7/0702.tex]: Say that you give it the name `2.2.a_1

File: exercises/APE7/0702.tex [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+\item $\pi_{2.2.b_1} = \frac{0.4\cdot 0.25}{0.6+0.4 \cdot 0.25} = \frac{1}{7}$ 
+\item $\pi_{2.3}     = 1 $
+\end{itemize}
+$\pi_{1.1}=\pi_{2.3} = 1$ because there is only one possible way to be at 1.1 (or 2.3) knowing that the information state is 1.1 (or 2.3).
+
+\paragraph{b.} The sequential value of player $i$ for the move $d$ and information state $s$ $$ U_i(d| s, (\tau, \pi) ) = \sum_{x \in Y_s} \pi_s(x) U_i((\tau_{-i}, d), x)$$
+Meaning that we will only care about nodes x with information $s$. \\
+We obtain : 
+\begin{itemize}
+\item $U_{1.1} (\text{a}_1) = \pi_{1.1} (0.9 \cdot 1 + 0.1 \cdot 8) = 2.6$
```
Comment (blegat) [inline exercises/APE7/0702.tex]: Write it `U_1(a_1|1.1)` to be consistent with Definition 4.22 of the course notes

File: exercises/APE7/0702.tex [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+\item $U_{1.1} (\text{b}_1) = 0.25 (0.9 \cdot 9 + 0.1 \cdot 1) + 0.75 (0.05 \cdot 9 + 0.95 \cdot 1) = 3.1$
+\item $U_{2.2} (\text{a}_2) = \pi_{2.2.a_1} \cdot -1 + \pi_{2.2.b_1} \cdot 0 = \frac{-6}{7}$
```
Comment (blegat) [inline exercises/APE7/0702.tex]: Why `\text{a}_2` and not `a_2` ? It is `a_2` in the problem statement

**Comment by anckaertv** [comment]
I will resolve those problems and then get back to you with a solution that contains less errors

File: exercises/APE7/0702.tex [hunk 8]
```diff
@@ -0,0 +1,89 @@
+\paragraph{a.} Let's first define what a belief vector is : for $\pi \in  \times_{s \in S} \Delta Y_s $, $i \in N$, $s \in S_i$, and $x \in Y_s$, $\pi_s(x)$ is the probability assigned by player $i$ to be at node $x$ given the fact that $i$ knows his information state $s$. ($X$ the nodes, $Y_s\in X$ is the set of nodes that have information $s$). 
+
+We can calculate it with Bayes formula $$ \pi_s(y) = \frac{P(y | \tau, x^0)}{\sum_{x \in Y_s} P(x | \tau, x^0)}. $$
+
+Here there is four information states: 1.1, 2.2 via $a_1$ (let's note it from now on $2.2.a_1$), 2.2 via $b_1$ (again, let's note it $2.2.b_1$) and 2.3:
```
Comment (blegat) [inline exercises/APE7/0702.tex]: 4 nodes and 3 information states, not 4 information states.

File: exercises/APE7/0702.tex [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+\begin{itemize}
+\item $\pi_{1.1}= 1$ 
+\item $\pi_{2.2}(2.2.a_1) = \frac{0.6}{0.6+0.4 \cdot 0.25} = \frac{6}{7}$
+\item $\pi_{2.2}(2.2.b_1)= \frac{0.4\cdot 0.25}{0.6+0.4 \cdot 0.25} = \frac{1}{7}$ 
+\item $\pi_{2.3}     = 1 $
+\end{itemize}
+$\pi_{1.1}=\pi_{2.3} = 1$ because there is only one possible way to be at 1.1 (or 2.3) knowing that the information state is 1.1 (or 2.3).
+
+\paragraph{b.} The sequential value of player $i$ for the move $d$ and information state $s$ $$ U_i(d| s, (\tau, \pi) ) = \sum_{x \in Y_s} \pi_s(x) U_i((\tau_{-i}, d), x)$$
+Meaning that we will only care about nodes x with information $s$.
```
Comment (blegat) [inline exercises/APE7/0702.tex]: x -> $x$

File: exercises/APE7/0702.tex [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+\item $\pi_{2.3}     = 1 $
+\end{itemize}
+$\pi_{1.1}=\pi_{2.3} = 1$ because there is only one possible way to be at 1.1 (or 2.3) knowing that the information state is 1.1 (or 2.3).
+
+\paragraph{b.} The sequential value of player $i$ for the move $d$ and information state $s$ $$ U_i(d| s, (\tau, \pi) ) = \sum_{x \in Y_s} \pi_s(x) U_i((\tau_{-i}, d), x)$$
+Meaning that we will only care about nodes x with information $s$. 
+
+We obtain : 
+\begin{itemize}
+\item $U_1(a_1|1.1) = \pi_{1.1} (0.9 \cdot 1 + 0.1 \cdot 8) = 2.6$
```
Comment (blegat) [inline exercises/APE7/0702.tex]: "0.9 \cdot 2", not "0.9 \cdot 1", the 2.6 seems correct though

File: exercises/APE7/0702.tex [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+    \item $\alpha$ to $a_1$, $1-\alpha$ to $b_1$
+    \item $\lambda$ to the state $2.2.a_1$ and $1-\lambda$ to the state $2.2.b_1$
+    \item $\beta$ to $a_2$ and $1-\beta$ to $b_2$
+    \item $\gamma$ to $c_2$ and $1-\gamma$ to $d_2$
+\end{itemize} 
+with each of them being between 0 and 1, representing the probability of playing each move ($\alpha$ represent the probability of player one to play $a_1$, and so on).
+
+We directly see that for the information state 2.3, the player 2 will always play $d_2$ having a payoff of 4, and will never play $c_2$ as it has a payoff of 0. For it to happen, $\gamma$ must be equal to 0. $d_2$ is thus part of the equilibria at the information state 2.3. 
+
+For the information state 2.2, we will test each support with information $s$ (a support with information $s$ is noted $D(s)$) and see if a sequential equilibria exist for this information state.
```
Comment (blegat) [inline exercises/APE7/0702.tex]: "a sequential equilibriUM existS"

File: exercises/APE7/0702.tex [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+    \end{align*} 
+    Let's equal those two expressions and we find that $\beta = \frac{7}{8}$. 
+    
+    We have thus found the final part of the sequential equilibria: for this equilibria, $\alpha=\frac{1}{6}$ (information state 1.1), $\beta =\frac{7}{8}$ (information state 2.2).
+\end{itemize}
+    
+The equilibria  ($\tau$ is the behavioural strategy and $\pi$ is the belief vector) is :
+\begin{align*}
+        [\tau, \pi ]&= \left[ \left(\tau_{1.1},\tau_{2.2},\tau_{2.3} \right), \left(\pi_{1.1}, \pi_{2.2} , \pi_{2.3}\right) \right]\\
+        &=\left[ \left(\frac{1}{6} a_1 + \frac{5}{6} b_1, \frac{7}{8} a_2 +\frac{1}{8} b_2, d_2\right) , \left( 1, (\frac{4}{9};\frac{5}{9}),1\right)\right]
```
Comment (blegat) [inline exercises/APE7/0702.tex]: Write 4/9 [2.2.a_1] + 5/9 [2.2.b_1], it is not clear which one is the first and which one the second

Comment (blegat) [inline exercises/APE7/0702.tex]: 1/6 a_1 -> 1/6 [a_1]

**Pull request merged**